packages/style/scss/iiif.scss

```diff
@@ -79,6 +79,7 @@
   .mirador-canvas-nav {
     background-color: rgba(255 255 255 / 95%);
     align-items: center;
+    height: auto !important;
```

Comment:
This is no longer needed I think (it was to fix a glitch, which we shouldn't have anymore)

Reply:
Gone.